{"name":"TypeError","message":"analytics.middleware is not a function","stack":"TypeError: analytics.middleware is not a function\n    at Object.<anonymous> (C:\\xampp\\htdocs\\myapp1\\vpn-flatform1\\server\\app.js:113:19)\n    at Module._compile (node:internal/modules/cjs/loader:1734:14)\n    at Object..js (node:internal/modules/cjs/loader:1899:10)\n    at Module.load (node:internal/modules/cjs/loader:1469:32)\n    at Module._load (node:internal/modules/cjs/loader:1286:12)\n    at TracingChannel.traceSync (node:diagnostics_channel:322:14)\n    at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)\n    at Module.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:152:5)\n    at node:internal/main/run_main_module:33:47","type":"uncaughtException","timestamp":"2025-06-03T09:15:42.699Z"}
{"name":"TypeError","message":"analytics.middleware is not a function","stack":"TypeError: analytics.middleware is not a function\n    at Object.<anonymous> (C:\\xampp\\htdocs\\myapp1\\vpn-flatform1\\server\\app.js:113:19)\n    at Module._compile (node:internal/modules/cjs/loader:1734:14)\n    at Object..js (node:internal/modules/cjs/loader:1899:10)\n    at Module.load (node:internal/modules/cjs/loader:1469:32)\n    at Module._load (node:internal/modules/cjs/loader:1286:12)\n    at TracingChannel.traceSync (node:diagnostics_channel:322:14)\n    at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)\n    at Module.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:152:5)\n    at node:internal/main/run_main_module:33:47","type":"uncaughtException","timestamp":"2025-06-03T09:17:57.161Z"}
{"name":"TypeError","message":"analytics.middleware is not a function","stack":"TypeError: analytics.middleware is not a function\n    at Object.<anonymous> (C:\\xampp\\htdocs\\myapp1\\vpn-flatform1\\server\\app.js:113:19)\n    at Module._compile (node:internal/modules/cjs/loader:1734:14)\n    at Object..js (node:internal/modules/cjs/loader:1899:10)\n    at Module.load (node:internal/modules/cjs/loader:1469:32)\n    at Module._load (node:internal/modules/cjs/loader:1286:12)\n    at TracingChannel.traceSync (node:diagnostics_channel:322:14)\n    at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)\n    at Module.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:152:5)\n    at node:internal/main/run_main_module:33:47","type":"uncaughtException","timestamp":"2025-06-03T09:19:21.582Z"}
{"name":"TypeError","message":"analytics.middleware is not a function","stack":"TypeError: analytics.middleware is not a function\n    at Object.<anonymous> (C:\\xampp\\htdocs\\myapp1\\vpn-flatform1\\server\\app.js:113:19)\n    at Module._compile (node:internal/modules/cjs/loader:1734:14)\n    at Object..js (node:internal/modules/cjs/loader:1899:10)\n    at Module.load (node:internal/modules/cjs/loader:1469:32)\n    at Module._load (node:internal/modules/cjs/loader:1286:12)\n    at TracingChannel.traceSync (node:diagnostics_channel:322:14)\n    at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)\n    at Module.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:152:5)\n    at node:internal/main/run_main_module:33:47","type":"uncaughtException","timestamp":"2025-06-03T09:21:13.525Z"}
{"name":"TypeError","message":"analytics.middleware is not a function","stack":"TypeError: analytics.middleware is not a function\n    at Object.<anonymous> (C:\\xampp\\htdocs\\myapp1\\vpn-flatform1\\server\\app.js:113:19)\n    at Module._compile (node:internal/modules/cjs/loader:1734:14)\n    at Object..js (node:internal/modules/cjs/loader:1899:10)\n    at Module.load (node:internal/modules/cjs/loader:1469:32)\n    at Module._load (node:internal/modules/cjs/loader:1286:12)\n    at TracingChannel.traceSync (node:diagnostics_channel:322:14)\n    at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)\n    at Module.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:152:5)\n    at node:internal/main/run_main_module:33:47","type":"uncaughtException","timestamp":"2025-06-03T10:07:12.373Z"}
{"name":"TypeError","message":"analytics.middleware is not a function","stack":"TypeError: analytics.middleware is not a function\n    at Object.<anonymous> (C:\\xampp\\htdocs\\myapp1\\vpn-flatform1\\server\\app.js:113:19)\n    at Module._compile (node:internal/modules/cjs/loader:1734:14)\n    at Object..js (node:internal/modules/cjs/loader:1899:10)\n    at Module.load (node:internal/modules/cjs/loader:1469:32)\n    at Module._load (node:internal/modules/cjs/loader:1286:12)\n    at TracingChannel.traceSync (node:diagnostics_channel:322:14)\n    at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)\n    at Module.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:152:5)\n    at node:internal/main/run_main_module:33:47","type":"uncaughtException","timestamp":"2025-06-03T10:09:53.865Z"}
{"name":"TypeError","message":"analytics.middleware is not a function","stack":"TypeError: analytics.middleware is not a function\n    at Object.<anonymous> (C:\\xampp\\htdocs\\myapp1\\vpn-flatform1\\server\\app.js:113:19)\n    at Module._compile (node:internal/modules/cjs/loader:1734:14)\n    at Object..js (node:internal/modules/cjs/loader:1899:10)\n    at Module.load (node:internal/modules/cjs/loader:1469:32)\n    at Module._load (node:internal/modules/cjs/loader:1286:12)\n    at TracingChannel.traceSync (node:diagnostics_channel:322:14)\n    at wrapModuleLoad (node:internal/modules/cjs/loader:235:24)\n    at Module.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:152:5)\n    at node:internal/main/run_main_module:33:47","type":"uncaughtException","timestamp":"2025-06-03T10:13:52.564Z"}
{"name":"UnhandledRejection","message":"logger.error is not a function","stack":"TypeError: logger.error is not a function\n    at startServer (C:\\xampp\\htdocs\\myapp1\\vpn-flatform1\\server\\app.js:303:16)","type":"unhandledRejection","timestamp":"2025-06-03T10:15:50.233Z"}
{"name":"TypeError","message":"logger.error is not a function","stack":"TypeError: logger.error is not a function\n    at process.<anonymous> (C:\\xampp\\htdocs\\myapp1\\vpn-flatform1\\server\\app.js:363:12)\n    at process.emit (node:events:519:35)\n    at emitUnhandledRejection (node:internal/process/promises:252:13)\n    at throwUnhandledRejectionsMode (node:internal/process/promises:388:19)\n    at processPromiseRejections (node:internal/process/promises:475:17)\n    at process.processTicksAndRejections (node:internal/process/task_queues:106:32)","type":"uncaughtException","timestamp":"2025-06-03T10:15:50.243Z"}
